lock.go

```diff
@@ -134,6 +118,29 @@ func (l *Lock) MarshalJSON() ([]byte, error) {
 	return buf.Bytes(), err
 }
 
+// TODO(carolynvs) this should be moved to gps
```

Comment:
It would only need to move to `gps` if the lock diffing implementation was moved to `gps` as it's used when doing the diff. Here's an issue to discuss if we really want to move it and how. #312